hips/utils/healpix.py

```diff
@@ -50,7 +53,32 @@ def boundaries(nside: int, pix: int, nest: bool=True) -> tuple:
         [( 264.375, -24.62431835), ( 258.75 , -30.        ),
          ( 264.375, -35.68533471), ( 270.   , -30.        )]>
     """
-
     boundary_coords = hp.boundaries(nside, pix, nest=nest)
     theta, phi = hp.vec2ang(np.transpose(boundary_coords))
     return theta, phi
+
+def compute_healpix_pixel_indices(wcs_geometry: WCSGeometry, nside: int) -> np.ndarray:
+    """Returns an array containing pixels corresponding to an image.
```

Comment:
> Returns an array containing pixels corresponding to an image.

I don't think this is quite accurate. Usually the return list of HEALPix pixels will contain pixels completely outside the image, no? (because you're doing a disk query and images don't correspond to disk regions. Suggest to change the description a bit to be more precise on that this function does.